backend/src/Logging.hs

```diff
@@ -0,0 +1,46 @@
+module Logging where
+
+import Control.Monad.Logger (Loc, LogLevel (..), LogSource, LogStr, defaultLoc)
+import System.Log.FastLogger
+  ( FormattedTime,
+    LogType' (..),
+    ToLogStr (..),
+    defaultBufSize,
+    newTimeCache,
+    newTimedFastLogger,
+    simpleTimeFormat,
+  )
+
+type Logger = Loc -> LogSource -> LogLevel -> LogStr -> IO ()
```

Comment:
This is the type expected by the [MonadLogger typeclass](https://hackage.haskell.org/package/monad-logger-0.3.40/docs/Control-Monad-Logger.html#g:1) we're using. We want to leverage this typeclass because it allows us to log from anywhere within our monad transformer stack, without having to manually lift the function into the actual logging monad (very convenient).

Reply:
In practice, we won't really use Loc or LogSource. But LogLevel is nice to use.